### PR TITLE
docs(implementations.md): add ramda-adjunct

### PR DESCRIPTION
**Pull request by char0n**
Ramda adjunct contains functional utils  for monadic types from monet, ramda-fantasy and contains reference implementation of Identity monad compliant with fantasy-land.

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ Here are a list of implementations that live in Fantasy Land:
 * [List in JS](https://github.com/PandaNoir/List-in-JS/) implements Setoid, Semigroup, Monoid, Functor, Applicative, Foldable, Traversable, Chain and Monad.
 * [Sanctuary](https://github.com/plaid/sanctuary) is a refuge from unsafe JavaScript. It provides FL-compatible Either and Maybe types.
 * [Fluture](https://github.com/Avaq/Fluture) is a high-performance monadic alternative to Promises.
+* [Ramda Adjunct](https://github.com/char0n/ramda-adjunct) is a community maintained extension of Ramda
```
Comment (davidchambers) [inline implementations.md]: I think there should be a hyphen (`-`) between _community_ and _maintained_: a _community-maintained_ extension.

**Comment by joneshf** [comment]
@davidchambers How would you feel about adopting something like overcommit: https://github.com/brigade/overcommit?

**Comment by davidchambers** [comment]
I find Git hooks quite annoying. I view committing as an action I should be able to perform regardless of whether tests are passing.

On the other hand, I'm a big believer in a [comprehensive `make lint`][1] (or equivalent), to be run by Travis (and which can also be run locally).


[1]: https://github.com/sanctuary-js/sanctuary/blob/v0.13.2/Makefile#L32-L57

**Comment by joneshf** [comment]
Sorry, I should have been clearer. I mean for the commit message checks: https://github.com/brigade/overcommit#commitmsg.

**Comment by davidchambers** [comment]
Actually it was me who was unclear:

> I view committing as an action I should be able to perform regardless of whether tests are passing.

I was using _tests_ to refer to anything which must be verified in order to get a :white_check_mark:.

Automating the validation of commit messages is worth doing if there are standards we wish to enforce. My preference, though, is not to tie this validation to the `git commit` action.

**Comment by char0n** [comment]
@davidchambers everything should be in order now. Thank you.

**Comment by Avaq** [comment]
There's a typo in the body of your commit message: *communit* -> *community*

File: implementations.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ Here are a list of implementations that live in Fantasy Land:
 * [List in JS](https://github.com/PandaNoir/List-in-JS/) implements Setoid, Semigroup, Monoid, Functor, Applicative, Foldable, Traversable, Chain and Monad.
 * [Sanctuary](https://github.com/plaid/sanctuary) is a refuge from unsafe JavaScript. It provides FL-compatible Either and Maybe types.
 * [Fluture](https://github.com/Avaq/Fluture) is a high-performance monadic alternative to Promises.
+* [Ramda Adjunct](https://github.com/char0n/ramda-adjunct) is a communit-maintained extension of Ramda
```
Comment (Avaq) [inline implementations.md]: Oh, the typo is also here.

**Comment by char0n** [comment]
@davidchambers all done. Sorry for the typos

**Pull request merged**